spyder/api/panel.py

```diff
@@ -116,7 +116,13 @@ def on_install(self, editor):
             self.setAttribute(Qt.WA_TransparentForMouseEvents)
 
     def paintEvent(self, event):
-        """Fills the panel background using QPalette."""
+        """
+        Fill the panel background using QPalette.
+
+        Notes:
+            - Please remember to extend this method in the child class to
+            paint the desired information of the panel.
```

Comment:
Given that we're trying to follow the [Numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) standard for new docstrings, the format of this section needs to rewritten as:

```
Notes
-----
Please remember to extend this method in the child class to paint the
desired information of the panel.
```

Please also remember that docstring lines should go up to 72 (not 79) characters (although we have not been enforcing that).